src/server/config/auth.js

```diff
@@ -1,11 +1,29 @@
-export function validateAccessTokenFacebook(accessToken) {
-  ///TODO : Validiate accesstoken from facebook API server.
+import request from 'request';
 
-  return false;
+export function crawlByAccessTokenFacebook(accessToken, callback) {
+  // Crawl user data from facebook by access token.
+  request.get('https://graph.facebook.com/me?fields=name,email,locale,timezone,verified&access_token='
+    + accessToken, function (error, response, userBody) {
+    if (!error && response.statusCode == 200) {
+      let result = JSON.parse(userBody);
+      // Crawl user profile_picture from facebook by access token.
+      request.get('https://graph.facebook.com/'
```

Comment:
Again, put the API reference in the comment, and declare these as const up top and use those const vars here.